pkg/overlay/pack.go

```diff
@@ -408,7 +408,7 @@ func generateLayer(config types.StackerConfig, oci casext.Engine, mutators []*mu
 		defer blob.Close()
 
 		if layerType.Type == "tar" {
-			desc, err = mutator.Add(context.Background(), mediaType, blob, history, mutate.GzipCompressor, nil)
+			desc, err = mutator.Add(context.Background(), mediaType, blob, history, mutate.GzipCompressor.WithOpt(mutate.GzipBlockSize(256<<12)), nil)
```

Comment:
Could we move the size to a const somewhere that's searchable, also with a comment indicating what it was previously and why we're changing it now?

Reply:
The commit msg/log captures that. But I can add a note.
Fixing the const.

Reply:
Fixed.